packages/patternfly-4/react-table/src/components/Table/base/evaluate-transforms.ts

```diff
@@ -1,16 +1,17 @@
 /**
- * evaluate-transforms.js
+ * evaluate-transforms.ts
  *
  * Forked from reactabular-table version 8.14.0
  * https://github.com/reactabular/reactabular/tree/v8.14.0/packages/reactabular-table/src
- * */
+ */
 import { isFunction } from 'lodash';
-import mergeProps from './merge-props';
+import { mergeProps } from './merge-props';
+import { transformsType, ExtraParamsType } from './types';
 
-function evaluateTransforms(transforms = [], value, extraParameters = {}) {
+export function evaluateTransforms(transforms:transformsType = [], value: string | object, extraParameters:ExtraParamsType = {}) {
   if (process.env.NODE_ENV !== 'production') {
     if (!transforms.every(isFunction)) {
-      throw new Error("All transforms weren't functions!", transforms);
+      throw new Error('All transforms weren\'t functions!');
```

Comment:
Why not display `transforms` as part of this error? Did TS complain there?

Reply:
yes, our version of TypeScript does not allow passing a second param to `ErrorConstructor`... 

![Screen Shot 2019-08-09 at 2 58 12 PM](https://user-images.githubusercontent.com/4237045/62802564-28a18900-bab6-11e9-8629-adc13db2a810.png)

Reply:
Ah weird, that makes sense though. 👍